File: js/stockPicker.js
exports.picker = function(prices) {
    let possiblePairs=[]
    let valuedPair=[0,0]
    for (i=0; i<prices.length; i++){
        // console.log(prices[i])
        for (n=i+1; n<prices.length;n++){
            if(prices[i]<prices[n]){
                possiblePairs.push([prices[i],prices[n]])
            }
        }
    }
//    console.log(possiblePairs)
    for (j=0;j<possiblePairs.length; j++){
        evaluator=[possiblePairs[j], (possiblePairs[j][1]-possiblePairs[j][0])]
        // console.log(evaluator)
        if (valuedPair[1]< evaluator[1]){
            valuedPair=evaluator
        }
    }
    return [prices.indexOf(valuedPair[0][0]), prices.indexOf(valuedPair[0][1])]
}
// picker([17,3,6,9,15,8,6,1,10])

File: python/stock_picker.py

def picker(prices):
    possible_pair=[]
    valued_pair= (0,0)
    for num in prices:
        for n in prices[prices.index(num):]:
            if num<n:
                possible_pair.append([num,n])
    #print(possible_pair)
    for pair in possible_pair:
        evaluator=(pair, (pair[1]-pair[0]))
        if valued_pair[1] < evaluator[1]:
            valued_pair=evaluator        
    #print(valued_pair)
    return [prices.index(valued_pair[0][0]),prices.index(valued_pair[0][1])]

#print(stock_picker([19,17,6,9,8,15,6,3,1]))